package.json

```diff
@@ -15,7 +15,7 @@
     "start:dev": "set DEBUG=http:* && nodemon src/index.js --exec babel-node",
     "test": "npm run db:setup && set DEBUG=http:* && cross-env NODE_ENV=test nyc mocha --timeout 120000 src/test/** --exit",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "seed": "cross-env NODE_ENV=test sequelize db:seed:all"
+    "seed": "cross-env NODE_ENV=test babel-node ./node_modules/.bin/sequelize db:seed:all"
```

Comment:
remove `cross-env NODE_ENV=test` from the seed script and set `cross-env NODE_ENV=test npm run seed`  in db:setup script or `cross-env NODE_ENV=test sequelize db:seed `

Reply:
Thanks 😀, done